engo_glfw.go

```diff
@@ -317,7 +323,8 @@ func SetVSync(enabled bool) {
 	}
 }
 
-func init() {
+// Initializes application
+func InitGl() {
 	runtime.LockOSThread()
 
 	Grave = Key(glfw.KeyGraveAccent)
```

Comment:
Yes but that is related to the other PR. I totally agree. I used the actual code.